db/schema.rb

```diff
@@ -1296,6 +1296,9 @@
     t.integer "following_orgs_count", default: 0, null: false
     t.integer "following_tags_count", default: 0, null: false
     t.integer "following_users_count", default: 0, null: false
+    t.string "forem_created_at"
+    t.string "forem_username"
+    t.datetime "github_created_at"
```

Comment:
I don't think this should be part of this PR. In fact, we removed all `<provider>_created_at` columns a few months back as part of https://github.com/forem/forem/issues/11414.

Reply:
Very likely this was added by mistake, I'll get rid of it. Thanks for flagging this!